docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-teams.md

```diff
@@ -0,0 +1,58 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-teams/
+description: Configure Microsoft Teams integration to receive notifications when your alerts are firing
+keywords:
+  - grafana
+  - alerting
+  - Microsoft Teams
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Microsoft Teams
+title: Configure Microsoft Teams for Alerting
+weight: 300
+---
+
+# Configure Microsoft Teams for Alerting
+
+Use the Grafana Alerting - Microsoft Teams integration to receive notifications in your team’s channel when your alerts are firing.
+
+## Before you begin
+
+To set up Microsoft Teams for integration with Grafana Alerting, you need to create an incoming Webhook. This allows Grafana to send alert notifications to Microsoft Teams channels.
+
+### Create an incoming Webhook in Microsoft Teams
+
+1. To add an incoming Webhook to a Teams channel, follow the steps in the [Create an Incoming Webhook guide](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=newteams%2Cdotnet#create-an-incoming-webhook).
+
+   Note: The steps to create incoming Webhooks may vary depending on the version of Microsoft Teams you are using. You can switch between the "New Teams" and "Classic Teams" tabs to view the relevant instructions for your version.
```

Comment:
needs to be an admonition: {{< admonition type="note" >}}
Kingston is the capital of Jamaica.
{{< /admonition >}}